Bazel/CMake: make cmake runnable from outside the workspace (#10953)
github/codeql: misc/bazel/cmake/README.md

This directory contains a Bazel to CMake generator intended mainly for IDE integration.

[`cmake.bzl`](./cmake.bzl) contains the Bazel side, with the `cmake_aspect` rule gathering the necessary information
from `cc_*` rules and `generate_cmake` translating that information into CMake commands. `generate_cmake` targets also

> depend on all files that are either generated or fetched from external repositories, so that Bazel will fill in those
> dependencies before letting CMake do the C/C++ compilation.

Reviewer: Do you mean:
```suggestion
depend on all files that are either generated or fetched from external repositories, Bazel will generate and fetch those
dependencies before letting CMake do the C/C++ compilation.
```
